docs/source/configuration/telemetry/overview.mdx

```diff
@@ -1,19 +1,19 @@
 ---
-title: Apollo Router Telemetry
+title: GraphOS Router Telemetry
```

Comment:
Available in both Router Core and GraphOS Router (for all occurrences on this page)